README.md

```diff
@@ -77,3 +77,33 @@ On some systems, `lean`/`leanpkg` will not work out of the box even if installed
 * Removed Windows installer... for now?
 * Merged `CARGO_HOME` and `RUSTUP_HOME`
 * Removed options to configure host triple
+
+# Build
```

Comment:
Can you adjust these instructions to https://github.com/leanprover/elan/issues/44#issuecomment-939251683 (env vars shouldn't be necessary anymore, acquiring openssl on Windows might be easier via `vcpkg`).

Reply:
Awesome, thanks for fixing that.  I updated the readme.